github-wiki-notify.php

```diff
@@ -44,18 +47,26 @@
 	exit(2);
 }
 
+// request repo-URL from git remote
+$remote = `git remote -v`;
+$repo = 'unknown';
+if (preg_match('/origin\s*(\S*)\s*\(fetch\)\n/', $remote, $match))
+{
+    $repo = $match[1];
+}
+
 $pullResult = `git pull 2>&1`;
-if (preg_match('/From github\.com:(.*)\n\s*([^\s]+)/', $pullResult, $match))
+if (preg_match('/^\S* ([a-z0-9]{7}\.\.[a-z0-9]{7})\n/', $pullResult, $match))
```

Comment:
What if you restore all the other code (remove `git remote -v` above and the changes inside this `if` block) and simply update this regex to support both the HTTPS and SSH formats? This regex would do it:

`'/From .*?github\.com[\/:](.*)\n\s*([^\s]+)/'`

So then `$repo` would be `awbush/test-wiki.wiki` and `$revs` would be `a4da57f..7e129dd` (in both examples below).

Reply:
With your notes I agree to give it a second try.
What is your preferred solution?
Should I add a fixing commit to this branch or should I start over a new one?
And which version number should I use at least?

Reply:
I like the idea of just updating the regex. It focuses on the issue: Add support for HTTPS remotes (while keeping support for legacy/SSH ones).

You can add a commit if that's easiest. I can squash merge into main branch.